src/app/Store/configStore.js

```diff
@@ -0,0 +1,13 @@
+import { createStore } from 'redux';
+import rootReducer from '../Reducers';
+
+export default function configureStore(initialState) {
```

Comment:
create the first instance of a store
pass in the reducer (which could be one reducer or multiple ones)
have the reducers hot reload when reducer files change